code/items/equipments/C4Equipment.cs

```diff
@@ -43,44 +43,26 @@ public override void Simulate(Client client)
 
             using (Prediction.Off())
             {
-                if (Input.Pressed(InputButton.Attack1)) //Drops C4
-                {
-                    (owner.Inventory as Inventory).DropEntity(this, typeof(C4Entity));
-                }
-                else if (Input.Pressed(InputButton.Attack2)) //Plants C4
+                if (Input.Pressed(InputButton.Attack1))
                 {
                     TraceResult placementTrace = Trace.Ray(Owner.EyePos, Owner.EyePos + Owner.EyeRot.Forward * PlaceDistance)
-                        .Ignore(owner)
-                        .UseHitboxes()
-                        .Run();
-                    if (!placementTrace.Hit)
+                       .Ignore(owner)
+                       .UseHitboxes()
+                       .Run();
+
+                    if (!placementTrace.Hit || !placementTrace.Entity.IsWorld)
                     {
                         return;
                     }
 
                     C4Entity bomb = new C4Entity();
-                    if (placementTrace.Entity.IsWorld)
-                    {
-                        bomb.PhysicsEnabled = false;
-                    }
-                    if (placementTrace.Entity is Prop prop)
-                    {
-                        bomb.CollisionGroup = CollisionGroup.Debris;
-                        bomb.Parent = prop;
-                    }
 
+                    bomb.PhysicsEnabled = false;
                     bomb.Position = placementTrace.EndPos;
                     bomb.WorldAng = placementTrace.Normal.EulerAngles;
                     bomb.Rotation = bomb.Rotation.RotateAroundAxis(Vector3.Right, -90);
                     bomb.Rotation = bomb.Rotation.RotateAroundAxis(Vector3.Up, 90);
 
-                    if (placementTrace.Entity is TTTPlayer player)
-                    {
-                        bomb.EnableSolidCollisions = false;
-                        bomb.AttachedBone = placementTrace.Bone;
```

Comment:
Are totally ditching the ability to place on players? Might be a fun off-by-default feature.

Reply:
Personally, I think it's a feature that adds too much uncertainty and wackiness to the C4. I think for now we should keep things simple, and then get a better feel for what things we can add in the future and add them in.